prioritization_fees/src/rpc_data.rs

```diff
@@ -23,6 +23,29 @@ pub struct PrioFeesStats {
     pub cu_consumed: TxAggregateStats,
 }
 
+impl PrioFeesStats {
+    pub fn get_percentile(&self, percentile: f32) -> Option<(u64, u64)> {
+        let index_tx = self
+            .by_tx_percentiles
+            .iter()
```

Comment:
could be simplified with this `a.iter().position(|&x| x == 2)`

https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.position